gluoncv/utils/metrics/tracking.py

```diff
@@ -1,7 +1,35 @@
 """ SiamRPN metrics """
 import numpy as np
-from colorama import Style, Fore
 
+def Iou(rect1, rect2):
```

Comment:
1. Where is it used?
2. Reasons for not using `F. contrib.box_iou` or `gluoncv.utils.bbox. bbox_iou`?

Reply:
1、It's used in dataset. Details at https://github.com/dmlc/gluon-cv/blob/master/gluoncv/data/tracking_data/track.py#L549
2、It's used in dataset and type is numpy, so not using _F. contrib.box_iou_
      and Iou input shape is (4,5,17,17) and  (4),_gluoncv.utils.bbox. bbox_iou_ input shape is (n,4),(n,4).It's not same,but it can adjust

Reply:
1、It's used in dataset. Details at https://github.com/dmlc/gluon-cv/blob/master/gluoncv/data/tracking_data/track.py#L549
2、It's used in dataset and type is numpy, so not using `F. contrib.box_iou`
and Iou input shape is (4,5,17,17) and (4),`gluoncv.utils.bbox. bbox_iou` input shape is (n,4),(n,4).It's not same,but it can adjust

Reply:
1、It's used in dataset. Details at https://github.com/dmlc/gluon-cv/blob/master/gluoncv/data/tracking_data/track.py#L549
2、It's used in dataset and type is numpy, so not using F. contrib.box_iou
and Iou input shape is (4,5,17,17) and (4),gluoncv.utils.bbox. bbox_iou input shape is (n,4),(n,4).It's not same,but it can adjust

Reply:
Let's re-use `gluoncv.utils.bbox.bbox_iou` to avoid duplicated efforts as well as fixes for potential bugs.

Reply:
ok, I have an idea, Detection's Iou in forward but tracking in dataset, Is this a reason to cost time?

Reply:
It's not about runtime efficiency. It is about the difficulty of maintenance. You can call `gluoncv.utils.bbox.bbox_iou` in your `Iou` function to compute the numbers. In this way, we only have to modify one place, if the iou computation goes wrong or an upgrade of iou computation is needed.

Reply:
sorry, I understand that. I mean `Tracking` train slower now. Iou is in dataset, per dataloader need cost time to calculate. if change Iou to forward or hybrid_forward like `Detection`. I don't know if it speed up

Reply:
You need to profile the code to figure out the most time-consuming part. Here's a tutorial and I highly encourage you to read and follow: https://mxnet.apache.org/api/python/docs/tutorials/performance/backend/profiler.html

Reply:
ok